homeassistant/components/thermostat/heat_control.py

```diff
@@ -40,6 +40,7 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     min_temp = util.convert(config.get(CONF_MIN_TEMP), float, None)
     max_temp = util.convert(config.get(CONF_MAX_TEMP), float, None)
     target_temp = util.convert(config.get(CONF_TARGET_TEMP), float, None)
+    tol_temp = config.get(CONF_TOL_TEMP, DEFAULT_TOL_TEMP)
```

Comment:
Use the `util.convert` method like `target_temp` on the line above. The last argument is the default value.